data/category-porn

```diff
@@ -1653,7 +1680,6 @@ ixxx.com
 iyottube.com
 jabcomix.com
 jable.org
-jable.tv
```

Comment:
This domain is already exciting in `jable`

Reply:
i see